src/components/SEO.js

```diff
@@ -5,233 +5,239 @@ import Helmet from "react-helmet";
+
+	const realPrefix = config.pathPrefix === "/" ? "" : config.pathPrefix;
+	const homeURL = `${config.siteUrl}${realPrefix}`;
+	const URL = `${homeURL}${postPath || ""}`;
+	const image = `${homeURL}${config.siteBanner}`;
+
+	if (article) {
+		const postMeta = postNode.frontmatter;
+		title = `${postMeta.title} | ${config.siteTitle}`;
+		if (postMeta.hasOwnProperty("description")) {
+			description = postMeta.description;
+		} else if (postNode.hasOwnProperty("excerpt")) {
+			description = postNode.excerpt;
+		}
+
+		if (postMeta.hasOwnProperty("keywords")) {
+			keywords = postNode.frontmatter.keywords.join();
+		}
+	}
+
+	// schema.org in JSONLD format
+	// https://developers.google.com/search/docs/guides/intro-structured-data
+	// You can fill out the 'author', 'creator' with more data or another type (e.g. 'Organization')
+
+	const schemaOrgWebPage = {
+		"@context": "http://schema.org",
+		"@type": "WebPage",
+		url: URL,
+		headline: config.siteHeadline,
+		inLanguage: config.siteLanguage,
+		mainEntityOfPage: URL,
+		description: config.siteDescription,
+		name: config.siteTitle,
+		author: {
+			"@type": "Person",
+			name: config.author
+		},
+		copyrightHolder: {
+			"@type": "Person",
+			name: config.author
+		},
+		copyrightYear: "2020",
+		creator: {
+			"@type": "Person",
+			name: config.author
+		},
+		publisher: {
+			"@type": "Person",
+			name: config.author
+		},
+		datePublished: "2019-01-07T10:30:00+01:00",
+		dateModified: buildTime,
+		image: {
+			"@type": "ImageObject",
+			url: image
+		}
+	};
+
+	// Initial breadcrumb list
+
+	const itemListElement = [
+		{
+			"@type": "ListItem",
+			item: {
+				"@id": homeURL,
+				name: "Home"
+			},
+			position: 1
+		},
+		{
+			"@type": "ListItem",
+			item: {
+				"@id": `${homeURL}/patterns`,
+				name: "Mask Patterns"
+			},
+			position: 2
+		},
+		{
+			"@type": "ListItem",
+			item: {
+				"@id": `${homeURL}/request-supplies`,
+				name: "Request for Assistance"
+			},
+			position: 2
+		},
+		{
+			"@type": "ListItem",
+			item: {
+				"@id": `${homeURL}/about-us`,
+				name: "About Us"
+			},
+			position: 2
+		},
+		{
+			"@type": "ListItem",
+			item: {
+				"@id": `${homeURL}/volunteer`,
+				name: "Join The Cause"
+			},
+			position: 2
+		}
+	];
+
+	let schemaArticle = null;
+
+	if (article) {
+		schemaArticle = {
+			"@context": "http://schema.org",
+			"@type": "Article",
+			author: {
+				"@type": "Person",
+				name: config.author
+			},
+			copyrightHolder: {
+				"@type": "Person",
+				name: config.author
+			},
+			copyrightYear:
+				postNode.parent && postNode.parent.hasOwnProperty("birthtime")
+					? postNode.parent.birthtime
+					: 2020,
+			creator: {
+				"@type": "Person",
+				name: config.author
+			},
+			publisher: {
+				"@type": "Organization",
+				name: config.author,
+				logo: {
+					"@type": "ImageObject",
+					url: `${homeURL}${config.siteLogo}`
+				}
+			},
+			//datePublished: postNode.parent.birthtime,
```

Comment:
@gvan12 I commented these out to prevent errors. I can't figure out how to adjust the query so this is included.